### PR TITLE
Spirillen patch 1

### PR DESCRIPTION
**Pull request by spirillen**
Commit for preview and properly a lot of git issues adding this on to the renamed branch

This commits

- issue templates
- README.md
- .gitignore

Please add any comments for enhancement

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,16 +1,59 @@
-## Hi there 👋
+## Phishing Database Project
 
-!WIP
+The Phishing database project aim's to provide up-to-date list of domains and URI there is hosting phishing contents.
```
Comment (g0d33p3rsec) [inline profile/README.md]: This sentence could use a couple of minor grammatical edits, I would adjust it to something like "The Phishing database project's aim is to provide an up-to-date list of domains and URIs that are hosting phishing contents."

Another possibility would be to omit the apostrophe entirely, "The Phishing database project aims..."

File: profile/README.md [hunk 2]
```diff
@@ -1,16 +1,59 @@
...
-**Here are some ideas to get you started:**
+### 🔗 Download the Full List of Data
 
-🙋‍♀️ A short introduction - what is your organization all about?
-🌈 Contribution guidelines - how can the community get involved?
-👩‍💻 Useful resources - where can the community find your docs? Is there anything else the community should know?
-🍿 Fun facts - what does your team eat for breakfast?
-🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
--->
+In the following table, you'll find links to our datasets and it's checksum.
```
Comment (g0d33p3rsec) [inline profile/README.md]: You can safely omit the apostrophe for "it's."

**Review by g0d33p3rsec — CHANGES_REQUESTED** [review]
Looks good.  I only see a couple of minor grammatical edits.

**Review by g0d33p3rsec — APPROVED** [review]

**Review by g0d33p3rsec — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (spirillen) [reply profile/README.md]: Accepted 👍🏻 thanks for the comment.

File: profile/README.md (hunk 2 above)
Comment (spirillen) [reply profile/README.md]: Accepted 👍🏻 thanks for the comment.

Both comments should be applied by now

**Comment by spirillen** [comment]
@funilrys and @mitchellkrogza any comments? so we can get on with the welcome page and close https://github.com/Phishing-Database/dev-center/issues/2

**Pull request closed without merging**